cmd/yurthub/app/start.go

```diff
@@ -148,14 +149,26 @@ func Run(ctx context.Context, cfg *config.YurtHubConfiguration) error {
 		}
 		gcMgr.Run()
 	} else {
-		klog.Infof("%d. disable gc manager for node %s because it is a cloud node", trace, cfg.NodeName)
+		klog.Infof("%d. disable gc manager for node %s because it is a cloud or local node", trace, cfg.NodeName)
 	}
 	trace++
 
 	klog.Infof("%d. new tenant sa manager", trace)
 	tenantMgr := tenant.New(cfg.TenantNs, cfg.SharedFactory, ctx.Done())
 	trace++
 
+	if cfg.WorkingMode == util.WorkingModeLocal {
```

Comment:
If the working mode is local, It seems that other functions is not needed to startup. because we only use yurthub to configure the local LB settings.

Reply:
done.